inspector/tasks.py

```diff
@@ -74,6 +74,30 @@ class StressNgSingleCore(lib.DockerTask):
     command: str = "-c \"nice -n -20 stress-ng --metrics --cpu 1 --cpu-method div16 -t 20 -Y /dev/stderr\""
 
 
+class StressNgLongRun(lib.DockerTask):
+    """
+    Runs the StressNg task for an extended time by running
+    stress-ng for 55 seconds in every minute, then sleeping until the
+    start of the next minute, repeated 1440 times.
+    This task is limited to a few instance types.
+    """
+
+    servers_only = [
```

Comment:
```suggestion
    servers_only = {
```
I would rather have a set.